README.md

```diff
@@ -86,16 +106,21 @@ Returns `result` for the first matching pattern. If there are no matching patter
 * `x && y` matches values which match both `x` and `y`
 * `x where condition` matches only if `condition` is true (`condition` may use any variables that occur earlier in the pattern eg `(x, y, z where x + y > z)`)
 * Anything else is treated as a constant and tested for equality
+* Expressions can be interpolated in as constants via standard interpolation syntax `$()`
```

Comment:
Maybe add a small example here as with the other cases? For example,  $(x) would already be sufficient.

Reply:
Yeah, I agree, I think just that -- `$(x)` -- is sufficient. thanks! Updated in latest commit.